projects/core/components/hosted-dropdown/hosted-dropdown.component.ts

```diff
@@ -59,7 +60,8 @@ export class TuiHostedDropdownComponent implements TuiFocusableElementAccessor {
     @ViewChild(TuiDropdownDirective)
     private readonly dropdownDirective?: TuiDropdownDirective;
 
-    private readonly manual$ = new BehaviorSubject(false);
+    /** TODO: rename in 4.0 */
+    readonly openChange = new BehaviorSubject(false);
```

Comment:
you can use `private` and skip `eslint-disable @typescript-eslint/member-ordering`

Reply:
Some things, like TUI_ARROW rely on `openChange` being public `EventEmitter`,

Reply:
sorry